test/Microsoft.TestPlatform.TestUtilities/TempDirectory.cs

```diff
@@ -26,26 +28,31 @@ public void Dispose()
     }
 
     public DirectoryInfo CreateDirectory(string dir)
-        => Directory.CreateDirectory(System.IO.Path.Combine(Path, dir));
+        => Directory.CreateDirectory(IO.Path.Combine(Path, dir));
+
+    public void CopyDirectory(string sourceDirectory, string targetDirectory)
+    {
+        CopyDirectory(new DirectoryInfo(sourceDirectory), new DirectoryInfo(targetDirectory));
+    }
 
-    public void CopyAll(DirectoryInfo source, DirectoryInfo target)
+    public void CopyDirectory(DirectoryInfo source, DirectoryInfo target)
```

Comment:
CopyDirectory is more obvious when you use this on strings, and I wanted to have both overloads with the same name.